jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -283,54 +288,38 @@ public void testBuildToDockerRegistry_dockerHubBaseImage()
   }
 
   @Test
-  public void testBuildToDockerDaemon()
+  public void testBuildToDockerDaemon_multipleTags()
       throws IOException, InterruptedException, ExecutionException, RegistryException,
-          CacheDirectoryCreationException {
-    buildDockerDaemonImage(
+          CacheDirectoryCreationException, InvalidImageReferenceException {
+    buildImage(
         ImageReference.of("gcr.io", "distroless/java", DISTROLESS_DIGEST),
-        ImageReference.of(null, "testdocker", null),
-        Collections.emptyList());
+        Containerizer.to(DockerDaemonImage.named("testdocker")),
+        Arrays.asList("testtag2", "testtag3"));
 
     progressChecker.checkCompletion();
 
-    assertDockerInspect("testdocker");
     assertLayerSize(7, "testdocker");
+    assertDockerInspect("testdocker");
```

Comment:
I know these methods were not introduced in this PR but since we are using a helper for the assert statements, could we make the name a bit more detailed? Something along the lines of `assertThatExpectedLayerSizeReturned` or `assertThatExpectedDockerInspectOutputReturned`.

Reply:
I wish I could, but `assertDockerInspect` checks a lot of image metadata, including labels, ports, environment variables, and history.